cf-net/cf-net.c

```diff
@@ -204,11 +206,25 @@ static void CFNetSetDefault(CFNetOptions *opts){
 
 static void CFNetInit()
 {
+    #ifdef __MINGW32__
+        InitializeWindows();
+        OpenNetwork();
+    #endif
     CryptoInitialize();
     LoadSecretKeys();
     cfnet_init(NULL, NULL);
 }
 
+static const char *CFNetGetWorkDir()
+{
+    //TODO: Can do this differently. I didn't want to depend on libpromises.
+    #ifdef __MINGW32__
+    return "C:\\Program Files\\Cfengine";
+    #else
+    return "/var/cfengine/bin";
+    #endif
+}
```

Comment:
This will break whoever uses custom `./configure` which is everybody besides CFEngine. I think it's better to depend on libpromises here too, and add a TODO for  moving `known_dirs.h` to libutils.